data/resources.yml

```diff
@@ -785,3 +785,11 @@
     - Research
   title: Roosevelt Institute
   url: https://rooseveltinstitute.org/program/climate-and-economic-transformation/
+- description: The Data Rescue Project is a clearinghouse for data rescue-related efforts and access points for US governmental data that are currently at risk. Founded in February 2025 by IASSIST, RDAP, and Data Curation Network members, they want to know what is happening in the community so that they can coordinate focus.
```

Comment:
```suggestion
- description: The Data Rescue Project is a clearinghouse for data rescue-related efforts for US governmental data that are currently at risk. Founded by IASSIST, RDAP, and Data Curation Network members, they want to know what is happening in the community so that they can coordinate focus.
```